test/integration/backward_compatible/parallel/map/MapLockTest.js

```diff
@@ -85,7 +86,10 @@ describe('MapLockTest', function () {
             }, [member, m]);
         }).then((c) => {
             clientTwo = c;
-            key = generateKeyOwnedBy(client, keyOwner);
+        }).then(() => {
+            return generateKeyOwnedBy(client, keyOwner);
```

Comment:
What about, instead of this, you write a new method, waitForNewPartitionTable, where in the method body, you loop from 0 to partition count, and check the owner of the partition, if it is equal to second member's uuid, return, if not wait a bit and try again (or, directly use assertTrueEventually).

My worry with the current implementation is that, we might be unlucky and 20 attempts might not be enough to generate a key owned by a second member

Reply:
ah ok I forgot the randomness here. sure.

Reply:
done